### ISSUE TITLE
Why not use server-sent events instead of page reloading?

### ISSUE THREAD

**Issue by cuzbog** [stuartlangridge/ubuntu-app-test#2]
Seems like server-sent events would fit for updating the status

**Comment by stuartlangridge** [comment]
Mainly because refreshing is easier and better supported, and we're not serving a hugely complicated page so there's no huge bandwidth benefit. However, I'd be happy to integrate a patch which progressively enhances this by checking if SSEs are supported and then not doing the reload. (Really, we should be using meta refreshes here rather than a JS `location.reload`, of course.)

**Comment by cuzbog** [comment]
Actually, SSEs are supported everywhere, except IE, Edge and Opera Mini, as always. To be precise, Chrome 9+, FF 6+, Opera 11+, Safari 5+. I think this compatibility is OK without a fallback, no?

**Comment by stuartlangridge** [comment]
Nope; we need a fallback, even if that's just "manually refresh the page". And I'm not as worried about compatibility as that adding SSEs here is quite a bit more work, although it will make the page more elegant! So, as noted, I'm more than happy to have SSE support added to the status page, and it'll make things better, but not to have that page _require_ SSE support to work.
